overlord/servicestate/quota_control_test.go

```diff
@@ -921,3 +939,227 @@ func (s *quotaControlSuite) TestCreateQuotaCreateQuotaConflict(c *C) {
 	_, err = servicestate.CreateQuota(st, "foo", "", []string{"test-snap2"}, 2*quantity.SizeGiB)
 	c.Assert(err, ErrorMatches, `quota group "foo" has "quota-control" change in progress`)
 }
+
+func (s *quotaControlSuite) TestMemoryCGroupDisabled(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	_memoryCGroupFile := servicestate.CGroupsFilePath
+	defer func() {
+		servicestate.SetCGroupsFilePath(_memoryCGroupFile)
+	}()
+
+	cgroupsPath := dirs.GlobalRootDir + "/proc/cgroups"
+	cgroupsFile, err := os.Create(cgroupsPath)
+	c.Assert(err, check.IsNil)
```

Comment:
Whoever edited this file before imported check using: `. "gopkg.in/check.v1"`, while this branch adds `"gopkg.in/check.v1"`, so we have the same package imported twice, but once with a package prefix (check.foo), and again without. Either this and other new tests need to be fixed like so:
```suggestion
	c.Assert(err, IsNil)
```
Or the existing tests need to be updated to use `check.` prefix. I would suggest to just update the tests added in this branch.

Reply:
Removed the check prefix, hence the extra import.